admin/ajax.php

```diff
@@ -43,23 +43,6 @@ function wpseo_set_option() {
 
 add_action( 'wp_ajax_wpseo_set_option', 'wpseo_set_option' );
 
-/**
- * Sets an option in the database to hide the index warning for a week.
- *
- * This function is used in AJAX calls and dies on exit.
- */
-function wpseo_set_indexation_remind() {
```

Comment:
In admin-global there is some logic that relies on this, we have to remove that as well.

See: https://github.com/Yoast/wordpress-seo/blob/P2-408-remove-remind-me-in-7-days/js/src/admin-global.js#L228

Reply:
I thought I removed that. Whoops.